app/src/main/java/com/musicplayer/database/musicPlaying/QueueDao.kt

```diff
@@ -9,6 +10,9 @@ interface QueueDao{
     @Query("SELECT * FROM queue limit 1")
     fun get():QueueWithSongsEntity
 
+    @Query("SELECT * FROM queue limit 1")
```

Comment:
Czym to query różni się od tego powyżej, czego nie rozumiem?

Reply:
Ten wyżej jest wykorzystywany w Write modelu(Commands), a ten przy Read modelu(Queries), lepiej było by trzymać je w osobnych DAO, ale narazie jest tylko jedna funkcja do odczytu, więc łatwiej było dodać ją tutaj. Wynika to z tego, żeby trzymać się zasad CQRS.
Dodatkowo przez to, że zwraca LiveData<..> to obserwator(czyli nasz widok) będzie notyfikowany za każdym razem jak coś się zmieni w QueueWithSongsEntity, nie trzeba z view modelu ciągle odpytywać bazy o stan kolejki, w skrócie mamy Push'a zamiast Pull'a.